### PR TITLE
contact

### PR DESCRIPTION
**Pull request by janmarkevin**

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -240,13 +240,29 @@ <h2 class="news__title">News</h2>
...
                 どんなにきれいなホームページも、成果につながらなければ価値にならない。
                 小さな力でも必ず役に立つ、仕事のため、マーケティングから制作、
-                プロモーションと常に世の中の流れを読み、最新技術に挑戦し続け成果につなげていきます。</p>
+                プロモーションと常に世の中の流れを読み、最新技術に挑戦し続け成果につなげていきます。
+            </p>
+        </div>
+    </section>
+    <section class="contacts">
+        <div class="contacts__btn-wrapper">
+            <button class="btn btn--contact btn--contact--grey">
```
Comment (sayeaah) [inline dist/index.html]: Kindly make it  ` btn--contact-grey`

File: dist/index.html [hunk 2]
```diff
@@ -240,13 +240,29 @@ <h2 class="news__title">News</h2>
...
+                <span class="btn__title--contact">03-5826-4652</span>
```
Comment (sayeaah) [inline dist/index.html]: Kindly add the base class before the modifier

File: dist/index.html [hunk 3]
```diff
@@ -240,13 +240,29 @@ <h2 class="news__title">News</h2>
...
+                <span class="btn__text--contact">午前9:00~午後 18:00</span>
```
Comment (sayeaah) [inline dist/index.html]: Kindly add the base class before the modifier

File: dist/index.html [hunk 4]
```diff
@@ -240,13 +240,29 @@ <h2 class="news__title">News</h2>
...
+                <img src="img/tele-white.png" alt="Telephone" class="btn__icon--contact">
```
Comment (sayeaah) [inline dist/index.html]: Kindly add the base class before the modifier

File: dist/index.html [hunk 5]
```diff
@@ -240,13 +240,29 @@ <h2 class="news__title">News</h2>
...
+            </button>
+            <button class="btn btn--contact">
+                <span class="btn__text--contact">メールでお問い合わせ</span>
```
Comment (sayeaah) [inline dist/index.html]: Kindly add the base class before the modifier

File: dist/index.html [hunk 6]
```diff
@@ -240,13 +240,29 @@ <h2 class="news__title">News</h2>
...
+                <span class="btn__title--contact">Mail Contact</span>
```
Comment (sayeaah) [inline dist/index.html]: Kindly add the base class before the modifier

File: dist/index.html [hunk 7]
```diff
@@ -240,13 +240,29 @@ <h2 class="news__title">News</h2>
...
+                <img src="img/mail-white.png" alt="Mail" class="btn__icon--contact">
```
Comment (sayeaah) [inline dist/index.html]: Kindly add the base class before the modifier

File: src/scss/components/_contacts.scss [hunk 8]
```diff
@@ -0,0 +1,10 @@
+.contacts {
+  padding: 50px 0 30px 0;
```
Comment (sayeaah) [inline src/scss/components/_contacts.scss]: You can use shorthand  `padding: 50px 0 30px;`

Comment (janmarkevin) [reply src/scss/components/_contacts.scss]: done

**Pull request merged**